Configuration/Settings.yaml

```diff
@@ -34,6 +34,15 @@ Neos:
           'Neos.Neos.UI:HostOnlyStyles':
             resource: '${"resource://" + Neos.Ui.StaticResources.compiledResourcePackage() + "/Public/Styles/HostOnlyStyles.css"}'
             position: 'start 900'
+          'WorkSans:200':
```

Comment:
@grebaldi @dimaip the referenced "brand-assets" is a package where the colors and fonts of Neos are defined, and yes they where used in the website back when I started it, this difference is ofc now kind of problematic. 

Was the change in font families meant to be just for the website or the whole brand of Neos? In the latter case ofc it makes sense to also change it in the UI doesnt it?